.travis.yml

```diff
@@ -0,0 +1,22 @@
+dist: xenial
+
+language: node_js
+
+node_js:
+  - node
```

Comment:
Is there a way to pick a more specific version?

Reply:
`node` provides the latest stable version. Do we want a different version?

Reply:
I think we should use what we get on our remote environment. That's why I picked 10.21 for the CircleCI. :-)

```
$ node --version
v10.21.0
```

Reply:
Makes sense! Will switch it over now